.travis.yml

```diff
@@ -20,4 +20,4 @@ deploy:
   github-token: $GITHUB_TOKEN
   keep-history: true
   on:
-    branch: master
```

Comment:
exclude this file from your commit

Reply:
How can i do this please? Sorry to bother you. I am a beginner..

Reply:
check [this](https://stackoverflow.com/questions/12481639/remove-files-from-git-commit) out, also you can use VS code text editor it simplifies a lot of things

Reply:
I have followed the instructions but i am not sure it works yet. 
Do i need to wait for my PR to be granted before i work on
another issue?

Reply:
1 have removed .travis from my commit. what else do i need to do...to move on please?

Reply:
You do not have to completely remove the file, you have to remove the changes added by you in the file. Add the file again by copying the code from the repository and push the changes.

Reply:
@Karantakalkar please have a look. I have done the suggested changes